motivation.md

```diff
@@ -12,7 +12,11 @@ project.
 <section class="slide">
 ## Why Git?
 
-Because many people are using GitHub to collaborating.
+Because many people are using GitHub to collaborate.
+Git provides a platform to objectively share the problem solving process.  
```

Comment:
This line sounds kind of meaningless, I think the text would be fine without it.